hack/addons/mindthegap-helm-registry/Dockerfile

```
@@ -3,10 +3,14 @@ FROM --platform=${BUILDPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERS
 # this gets called by goreleaser so the copy source has to be the path relative to the repo root.
 RUN --mount=source=./hack/addons/mindthegap-helm-registry/repos.yaml,target=/repos.yaml \
     ["/ko-app/mindthegap", "create", "bundle", "--helm-charts-file=/repos.yaml", "--output-file=/tmp/helm-charts.tar"]
+# TODO remove me as soon as its not needed to hold multiple versions of helm charts
+FROM ghcr.io/nutanix-cloud-native/caren-helm-reg:v0.14.7 as carenprevious
 
 FROM --platform=${TARGETPLATFORM} alpine:3.20.3
 ARG VERSION
 COPY --from=bundle_builder /tmp/helm-charts.tar /charts/helm-charts-${VERSION}.tar
 COPY --from=bundle_builder /ko-app/mindthegap /usr/bin/mindthegap
```

Realised this can be simplified btw:

```suggestion
FROM --platform=${TARGETPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERSION}
ARG VERSION
COPY --from=bundle_builder /tmp/helm-charts.tar /charts/helm-charts-${VERSION}.tar
```

---

Don't we need alpine because mindthegap container doesn't have the cp utility?

---

What do you need cp for? `COPY` in a dockerfile is used by docker to construct the image filesystem, cp in the image is not used.

---

we use it here https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/c6307ea157bad8546880d0137bdfdff577277d25/charts/cluster-api-runtime-extensions-nutanix/templates/helm-repository.yaml#L66